app/routes/home.js

```diff
@@ -10,18 +15,26 @@ export default Route.extend({
     },
   },
 
-  model({ page, tag }) {
+  model({ feed, page, tag }) {
     const NUMBER_OF_ARTICLES = 10;
-    const offset = (parseInt(page, 10) - 1) * NUMBER_OF_ARTICLES;
-    return this.store
-      .query('article', {
-        limit: NUMBER_OF_ARTICLES,
-        offset,
-        tag,
-      })
-      .then(articles => {
-        articles.meta.pageSize = NUMBER_OF_ARTICLES;
-        return articles;
+    if (feed === 'your') {
```

Comment:
We now fork depending on whether we're retrieving `Your Feed` vs `Global feed`/Tag  search. The `Your feed` API is at a different URI than the Global Feed/tag one